rosdep/base.yaml

```diff
@@ -310,6 +310,27 @@ boost:
     xenial: [libboost-all-dev]
     yakkety: [libboost-all-dev]
     zesty: [libboost-all-dev]
+boost-program-options:
+  ubuntu:
+    bionic: [libboost-program-options1.65.1]
+    precise: [libboost-program-options1.46.1]
+    trusty: [libboost-program-options1.54.0]
+boost-program-options-dev:
+  ubuntu: [libboost-program-options-dev]
+boost-random:
+  debian:
+    jessie: [libboost-random1.55.0]
+    stretch: [libboost-random1.62.0]
+    wheezy: [libboost-random1.49.0]
+  fedora: [boost-random]
+  ubuntu:
+    bionic: [libboost-random1.65.1]
+    precise: [libboost-random1.46.1]
+    trusty: [libboost-random1.54.0]
+boost-random-dev:
```

Comment:
Please don't change an existing rule. Especially a low level dependency like boost. That will break anyone who is using the rule.